src/App/Services/Tamara_Client.php

```diff
@@ -54,6 +75,467 @@ public function reinit_tamara_client( $api_token, $api_url = 'https://api.tamara
 		static::$instance->api_client = $client;
 	}
 
+	/**
+	 * Create Tamara Checkout session, if failed, put errors to `wc_notices`
+	 *
+	 * @param $wc_order_id
+	 *
+	 * @return array|bool
+	 * @throws \Exception
+	 */
+	public function tamara_checkout_session( $wc_order_id ) {
+		$wc_order = wc_get_order( $wc_order_id );
+		$instalment_period = 3;
+		$checkout_payment_type = 'PAY_BY_INSTALMENTS';
+		try {
+			$checkout_response = $this->create_tamara_checkout_session(
+				$wc_order,
+				$checkout_payment_type,
+				$instalment_period
+			);
+		} catch ( Exception $tamaraCheckoutException ) {
+			$errorMessage = $this->tamara_checkout_wp_plugin->_t( 'Tamara Service unavailable! Please try again later.' );
```

Comment:
You should create a method for this `_t` and on that method, you use `Tamara_Checkout_WP_Plugin::wp_app_instance()->_()`